src/components/breadcrumbs/_macro.spec.js

```diff
@@ -5,125 +5,130 @@ import * as cheerio from 'cheerio';
-
-        const testValuesA = mapAll($('.ons-breadcrumbs__link'), (node) => node.attr('data-a'));
-        expect(testValuesA).toEqual(['123', '789']);
-        const testValuesB = mapAll($('.ons-breadcrumbs__link'), (node) => node.attr('data-b'));
-        expect(testValuesB).toEqual(['456', 'ABC']);
+    describe('GIVEN: Params: id', () => {
+        describe('WHEN: id is provided', () => {
+            const $ = cheerio.load(
+                renderComponent('breadcrumbs', {
+                    ...EXAMPLE_BREADCRUMBS_REQUIRED_PARAMS,
+                    id: 'example-breadcrumbs',
+                }),
+            );
+            test('THEN: renders with provided id', () => {
+                expect($('.ons-breadcrumbs').attr('id')).toBe('example-breadcrumbs');
+            });
+        });
     });
-
-    it('has a "chevron" icon for each breadcrumb item', () => {
-        const faker = templateFaker();
-        const iconsSpy = faker.spy('icon');
-
-        faker.renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_MINIMAL);
-
-        const iconTypes = iconsSpy.occurrences.map((occurrence) => occurrence.iconType);
-        expect(iconTypes).toEqual(['chevron', 'chevron']);
+    describe('GIVEN: Params: itemsList', () => {
+        describe('WHEN: itemClasses param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item with provided style classes', () => {
+                expect($('.ons-breadcrumbs__item:first').hasClass('item-extra-class')).toBe(true);
+                expect($('.ons-breadcrumbs__item:first').hasClass('item-another-extra-class')).toBe(true);
+            });
+        });
+        describe('WHEN: linkClasses param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders link with provided style classes', () => {
+                expect($('.ons-breadcrumbs__link').hasClass('link-extra-class')).toBe(true);
+                expect($('.ons-breadcrumbs__link').hasClass('link-another-extra-class')).toBe(true);
+            });
+        });
+        describe('WHEN: id param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item with provided id', () => {
+                expect($('.ons-breadcrumbs__link').attr('id')).toBe('first-breadcrumb');
+            });
+        });
+        describe('WHEN: url param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item with provided url link', () => {
+                const url = mapAll($('.ons-breadcrumbs__link'), (node) => node.attr('href'));
+                expect(url).toEqual(['https://example.com/']);
```

Comment:
This is also only testing the first breadcrumb

Reply:
it didn't really make sense to me why the original was checking both to be honest - it didn't check both id params so it wasn't consistent - and if we want to tests for multiples then i would put that in a new set of givens, otherwise it's testing different things at the same time